data/frame.go

```diff
@@ -202,13 +202,21 @@ func (f *Frame) CopyAt(fieldIdx int, rowIdx int) interface{} {
 	return f.Fields[fieldIdx].vector.CopyAt(rowIdx)
 }
 
-// Set set the val to the specified fieldIdx and rowIdx.
+// Set sets the val to the specified fieldIdx and rowIdx.
 // It will panic if either the fieldIdx or rowIdx are out of range or
 // if the underlying type of val does not match the element type of the Field.
 func (f *Frame) Set(fieldIdx int, rowIdx int, val interface{}) {
 	f.Fields[fieldIdx].vector.Set(rowIdx, val)
 }
 
+// SetConcreateAt sets the val to the specified fieldIdx and rowIdx.
```

Comment:
This wording looks more correct to me, assuming that the method sets the value at (fieldIdx, rowIdx).
```suggestion
// SetConcreateAt sets the value of the specified fieldIdx and rowIdx.
```

Reply:
actually it's the opposite: the `val` is set at the specified `fieldIdx` and `rowIdx`
I think that @kylebrandt suggestion makes it clearer

Reply:
I don't think it would be the opposite, "of" and "at" to me sounds as if it means the same thing here. "to" would mean something else.